integration/tests/basic-runtime.ts

```diff
@@ -0,0 +1,97 @@
+// tslint:disable-next-line: match-default-export-name no-implicit-dependencies
+// tslint:disable-next-line: match-default-export-name
+import ApolloClient, { gql } from 'apollo-boost';
+import test, { ExecutionContext } from 'ava';
+import { unlinkSync } from 'fs';
+import { TestxServer } from 'graphql-testx';
+import { InMemoryDatabase } from 'graphql-testx/dist/src/InMemoryDatabase';
+import fetch from "node-fetch";
+import { migrateDB } from '../../packages/graphql-migrations';
+
+// tslint:disable-next-line: no-require-imports
+import * as Knex from 'knex';
+import { schema as basicSchema } from '../schemas/basic.graphql';
+import { schema as relationsSchema } from '../schemas/relations-schema.graphql';
+
+const dbOptions = {
+  client: "sqlite3",
+  connection: { filename: "./test.sql" }
+};
+
+let database;
+
+test.beforeEach(() => {
+  unlinkSync("./test.sql")
+
+  const knex = Knex(dbOptions);
+  database = new InMemoryDatabase(knex);
+})
+
+test.serial('Graphback runtime end to end', async (t: ExecutionContext) => {
+  const server = await createServer(basicSchema);
+  await graphbackRuntimeWorkflow(server, t);
+});
+
+test.serial('Graphback runtime relationships - generic OneToMany', async (t: ExecutionContext) => {
+  const server = await createServer(relationsSchema);
+  // TODO relationships not supported in queries and runtime
+  await graphbackRuntimeWorkflow(server, t);
+});
+
+async function graphbackRuntimeWorkflow(server: TestxServer, t: ExecutionContext<unknown>) {
+  // Test migrations
+  const migration = await migrateDB(dbOptions, basicSchema);
+  const dbSchema = await server.getDatabaseSchema();
```

Comment:
@wtrocki I had this idea of adding more information to the database schema, like the type (varchar, int, float, ..) and things like NOT NULL, UNIQUE, INDEX, ... would this improve your tests?

Reply:
I think we are ok. We have that info from migration. This one is only because info from migration is soo big that we want to simply see table structure in very simple model for testing and verification purposes.